tests/functional/test_model_completeness.py

```diff
@@ -34,15 +38,53 @@ def test_paginators_and_waiters_are_not_lost_in_new_version(
 ):
     # Make sure if a paginator and/or waiter exists in previous version,
     # there will be a successor existing in latest version.
-    loader = Loader()
     try:
-        loader.load_service_model(service_name, type_name, previous_version)
+        LOADER.load_service_model(service_name, type_name, previous_version)
     except DataNotFoundError:
         pass
     else:
         try:
-            loader.load_service_model(service_name, type_name, latest_version)
+            LOADER.load_service_model(service_name, type_name, latest_version)
         except DataNotFoundError as e:
             raise AssertionError(
                 f"{type_name} must exist for {service_name}: {e}"
             )
+
+
+def _endpoint_rule_set_cases():
+    for service_name in AVAILABLE_SERVICES:
+        versions = LOADER.list_api_versions(service_name, 'service-2')
+        for version in versions:
+            yield service_name, version
+
+
+@pytest.mark.parametrize(
+    "service_name, version",
```

Comment:
very minor nit and i don't think its causing any issues, but the docs don't have a space between the comma and next variable so it should probably be `"service_name,version"`

Reply:
I did a quick search in the repo for `@pytest.mark.parametrize` and found that all cases with multiple parameters are formatted `"param1, param2, ..."`. Should we keep this consistency or follow the docs. Im fine with either, just wanted to bring that up.

Reply:
I'd keep it as is then. Its more readable with the space and if its all over our repo then I'm confident it works fine.